fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergCatalog.java

```diff
@@ -130,11 +131,11 @@ default StarRocksIcebergTableScan getTableScan(Table table, StarRocksIcebergTabl
                 srScanContext);
     }
 
-    default String defaultTableLocation(String dbName, String tableName) {
+    default String defaultTableLocation(Namespace ns, String tableName) {
```

Comment:
Why only modify params of these two methods?

Reply:
Because these two functions only used in rest-catalog, change it's signature will not affect other's code.